packages/make-styles/package.json

```diff
@@ -17,8 +17,10 @@
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "test": "just-scripts test",
-    "test:watch": "just-scripts jest-watch"
+    "test": "jest",
+    "test:watch": "just-scripts jest-watch",
```

Comment:
bad copy paste? 
```suggestion

```

Reply:
It looks that "test:watch" is not handled by a generator, I deleted this line manually as it's a single package in converged that has such script.

Reply:
gotcha. you found another bug! nicely done 💪

Reply:
https://github.com/microsoft/fluentui/issues/18742